blazor/mention/customization.md

```diff
@@ -0,0 +1,60 @@
+---
+layout: post
+title: Customization in Blazor Mention Component | Syncfusion
+description: Checkout and learn here all about customization in Syncfusion Blazor Mention component and much more.
+platform: Blazor
+control: Mention
+documentation: ug
+---
+
+# Customization in Blazor Mention Component
+
+## Show or hide mention character
+
+You can show the mention character as the prefix of the selected item in the Mention component by using the `ShowMentionChar` property. By default, the `ShowMentionChar` property is set to `false`, which means that the mention character will not be displayed as part of the selected mention item.
+
+You can set the `ShowMentionChar` property to `true` to display the mention character as the prefix of the selected mention item. This can be useful in cases where you want to clearly differentiate between the selected mention item and the rest of the text in the Mention component.
+
+{% highlight razor %}
+
+{% include_relative code-snippet/show-mention-char.razor %}
+
+{% endhighlight %}
+
+![Blazor Mention with show or hide mention character](./images/blazor-mention-show-mention-char.png)
+
+## Adding the suffix character after selection
+
+The `SuffixText` property in the Mention component allows you to specify a string that should be appended to the end of the selected mention item when it is inserted into the input field. You can use this property to add a space or a new line after the mention, or any other string that you want to include.
```

Comment:
The [SuffixText](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.DropDowns.SfMention-1.html#Syncfusion_Blazor_DropDowns_SfMention_1_SuffixText) property in the Mention component allows you to specify a string that should be appended to the end of the selected mention item when it is inserted into the input field. You can use this property to add a `space` or a `new line` after the mention, or any other string that you want to include.

Reply:
Addressed review comments